lib/ansible/plugins/lookup/env.py

```diff
@@ -30,11 +30,11 @@
 EXAMPLES = """
 - name: Basic usage
   debug:
-    msg: "'{{ lookup('env', 'HOME') }}' is the HOME environment variable."
+    msg: "'{{ lookup('env', 'HOME') }}'" is the HOME environment variable."
```

Comment:
These changes appear to be incorrect. The `"` is terminated at the end of the line, due to YAML formatting requirements.  This actually causes this to become unparsable YAML (`ParserError: while parsing a block mapping`)

The output (as it currently exists) would look something like:

```
'/home/sivel' is the HOME environment variable
```

What problem were you attempting to resolve?